torch/_inductor/compile_fx.py

```diff
@@ -799,17 +799,19 @@ def get_input_idxs_to_check(
     inputs: Union[List[torch.Tensor], Sequence[int]],
     static_input_idxs: Sequence[int],
 ) -> Sequence[int]:
-    def is_aligned(storage_offset, dtype):
+    def is_aligned(input):
+        # See Note: [Input Alignment handling in Inductor]
+        if input._is_view() and not config.assume_aligned_inputs:
+            return False
```

Comment:
It's a little surprising to me that we're rechecking `_is_view()` here.. 

In our initial compilation we are determining which inputs we want to assume alignment of, and we take into account is_view() as a heuristic there. 

I would expect that we look up that list to check alignment instead of checking `is_view` here

Reply:
Is your suggestion to look at the list from GraphLowering? I feel like that's a bit hard because there's a layer of code caching in between, and it deals with variables in the GraphLowering fx graph.

Would it be fine instead to put this logic in some util function that we check twice (once here, and once in GraphLowering)? Or is there additional reason that we really want to use one list and only perform the check once?